src/main/kotlin/no/nav/syfo/dialogmote/DialogmoteService.kt

```diff
@@ -515,6 +523,23 @@ class DialogmoteService(
         }
     }
 
+    fun getArbeidstakerBrevFromUuid(
+        brevUuid: UUID
+    ): ArbeidstakerBrev {
+        val motedeltakerArbeidstakerVarsel = dialogmotedeltakerService.getDialogmoteDeltakerArbeidstakerVarselList(
+            uuid = brevUuid
+        ).firstOrNull()
+
+        val referat = getReferat(
+            referatUUID = brevUuid
+        )
+
+        if (motedeltakerArbeidstakerVarsel == null && referat == null) {
+            throw IllegalArgumentException("No Brev found for PersonIdent with uuid=$brevUuid")
```

Comment:
Stusset litt over "PersonIdent" i feilmeldingen her (skjønner at den kommer i kontekst av at denne metoden kalles fra API-et, men det trenger jo ikke alltid være tilfelle). Kanskje bedre med `"No Brev found for arbeidstaker with uuid=$brevUuid" ` ?